stdlib/public/core/DebuggerSupport.swift

```diff
@@ -15,15 +15,23 @@ import SwiftShims
 // Macros are disabled when Swift is built without swift-syntax.
 #if $Macros && hasAttribute(attached)
 
-/// Converts description definitions to a debugger type summary.
+/// Converts description definitions to a debugger Type Summary.
 ///
-/// This macro converts compatible `debugDescription` (or `description`)
-/// implementations to a debugger type summary. This improves debugging in
-/// situations where expression evaluation is not performed, such as the
-/// variable list of an IDE.
+/// This macro converts compatible description implementations written in Swift
+/// to a debugger format known as a Type Summary. A Type Summary is the
```

Comment:
... `which is understood by LLDB.`
I don't think there is any value in skirting around the fact that this aimed at LLDB. LLDB is **the** Swift debugger, so that's nothing we need to hide either.

Reply:
I've updated it.